pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/model/response/PICSUREResponse.java

```diff
@@ -0,0 +1,83 @@
+package edu.harvard.hms.dbmi.avillach.auth.model.response;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import java.util.HashMap;
+import java.util.Map;
+
+public class PICSUREResponse {
```

Comment:
I dislike the `ResponseEntity<?>` pattern you have in your controllers because the code doesn't tell me what's getting returned. In some cases, it's hard to fix because the type of the response changes depending on the code. In most cases though, we can do better. Here's a PR: https://github.com/hms-dbmi/pic-sure-auth-microapp/pull/171/files#diff-3c7325884d2e98b9fe82c38d7d85bbf638399aec2d4e2b37e23c636a4762dee5